.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "search/pybinds/pybind11"]
-	path = search/pybinds/pybind11
-	url = https://github.com/pybind/pybind11.git
+[submodule "lib/pybind11"]
+	path = lib/pybind11
+	url = git@github.com:pybind/pybind11.git
```

Comment:
I'm curious what the difference is between these two URLs? The new URL is giving me a "can't verify host" type warning message whereas the old URL works fine.

Reply:
This was a mistake on my part, I had an very old default git that didn't support `git mv` of submodules so I had to edit by hand. Returned to `https`